src/components/Dashboard.js

```diff
@@ -1,9 +1,121 @@
-import React from 'react'
+import React, {useState, useEffect} from "react";
+import * as yup from 'yup'
+import {Link} from "react-router-dom";
+import axios from 'axios'
+import DashboardSearchForm from './DashboardSearch'
+
+const initialFormValues = {
```

Comment:
Feel like we are going to need like the name title category source ingredients and instructions maybe.... just maybe